db/migrate/20191008124157_add_daily_summary_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDailySummaryToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :daily_summary, :boolean, default: false
```

Comment:
Maybe we should add `null: false` as well - just so you don't have to fill your head with null options.

Reply:
Great suggestion! Doing changes.